spark/src/main/resources/R/zeppelin_sparkr.R

```diff
@@ -31,7 +31,7 @@ print(paste("LibPath ", libPath))
 library(SparkR)
 
 
-SparkR:::connectBackend("localhost", port, 6000)
+SparkR:::connectBackend("localhost", port)
```

Comment:
This is due to sparkR api change

https://github.com/apache/spark/blob/master/R/pkg/R/client.R#L22